code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/ProjectResourceManager.java

```diff
@@ -84,7 +86,7 @@ public Object listInternal(SchemaFactory schemaFactory, String type, Map<Object,
                     throw new ClientVisibleException(ResponseCodes.NOT_FOUND);
                 }
             }
-            policy.grantObjectAccess(project);
+            giveProjectAccess(project);
```

Comment:
Having 2 arguments doesn't make it look complicated :) Plus we make it more simple by avoiding unnecessary calls.